### PR TITLE
yandex tracking script in head tag

### PR DESCRIPTION
**Pull request by Maxstgt**
smal fix
Screenshot from 2020-03-06 15-09-28

### CODE REVIEW

File: apps/catalog/catalog/src/index.html [hunk 1]
```diff
@@ -1,19 +1,22 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
```
Comment (lsenta) [inline apps/catalog/catalog/src/index.html]: What is happening here? It looks like all you did was to change the indentation

Comment (Maxstgt) [reply apps/catalog/catalog/src/index.html]: yes, I was trying to hard code the script tag to see if it is working. Shouldn't have pushed it to the PR!

**Comment by lsenta** [comment]
What's the rationale for this patch? The PR description is only "small fix" and there's no ticket associated.

File: apps/catalog/catalog/src/index.html (hunk 1 above)
Comment (lsenta) [reply apps/catalog/catalog/src/index.html]: Remove it, `git reset HEAD~2 -- path/to/filename.html`, `git add -u` `git commit --amend`

**Comment by Maxstgt** [comment]
> What's the rationale for this patch? The PR description is only "small fix" and there's no ticket associated.

The problem was, that we used a npm package for connecting our app with yandex metrika, but this package wasnt maintained anymore and was simply not working. So I did it by myself. And there is no issue about it. Vincent was asking me to do it.

File: apps/catalog/catalog/src/index.html (hunk 1 above)
Comment (Maxstgt) [reply apps/catalog/catalog/src/index.html]: thanks

**Pull request merged**